composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "require": {
-        "php": "~5.6.12",
+        "php": "^7.2",
```

Comment:
We should probably use `~` so we don't get accidentally upgraded to the latest 7.x.

Reply:
Ok. I'll set it back to ~.